BFWControls/Modules/Carousel/Controller/CarouselViewController.swift

```diff
@@ -142,6 +146,14 @@ open class CarouselViewController: UICollectionViewController {
         }
     }
     
+    fileprivate func showBounceWhenViewLoaded() {
```

Comment:
Should we perhaps just call this `showBounce()` since there's nothing in this function that checks that the view has loaded?